### DISCUSSION TITLE
[Enhancement] [color-picker] color-picker中输入框输入色值非正常情况展示优化

### DISCUSSION THREAD

**Discussion by SeeIknow** [element-plus/element-plus#16829] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `color-picker`

## Description

输入框能否支持输入事件callback或者优化输入框内容展示 用于处理非正常要求输入内容；
如正常输入：rgb(1,1,1) 非正常输入：rgb(1,1,1,1,1,11,1,1,1)  结果还是rgb(1,1,1)但是输入框展示还是rgb(1,1,1,1,1,11,1,1,1);
展示体验差 引起用户误会；

目前我们无法做到处理异常输入值及展示情况

能否优化下：
1.展示与实际值对应
2.提供事件 我们自己处理

### Additional comments

_(empty)_

**Comment by warmthsea** [comment]
可以提供详细细节么？
比如：playground + 截图
